apps/webhook/nest-cli.json

```diff
@@ -2,6 +2,25 @@
   "collection": "@nestjs/schematics",
   "sourceRoot": "src",
   "compilerOptions": {
-    "assets": [".env.test", ".env.development"]
+    "assets": [
+      {
+        "include": ".env",
+        "outDir": "dist/src"
+      },
+      {
+        "include": ".env.development",
+        "outDir": "dist/src"
+      },
+      {
+        "include": ".env.test",
+        "outDir": "dist/src"
+      },
+      {
+        "include": ".env.production",
+        "outDir": "dist/src"
+      }
+    ],
+    "plugins": [],
+    "webpack": true
```

Comment:
Switch webhook to work in similair flow to worker to align with how we run the services